modules/core/src/lib/deck.js

```diff
@@ -766,6 +783,40 @@ export default class Deck {
       mode: 'hover'
     });
   }
+
+  _getFrameStats() {
+    this.stats.get('frameRate').timeEnd();
+    this.stats.get('frameRate').timeStart();
+
+    // Get individual stats from luma.gl so reset works
+    const animationLoopStats = this.animationLoop.stats;
+    this.stats.get('GPU Time').addTime(animationLoopStats.get('GPU Time').lastTiming);
+    this.stats.get('CPU Time').addTime(animationLoopStats.get('CPU Time').lastTiming);
+  }
+
+  _getMetrics() {
+    this.metrics.fps = this.stats.get('frameRate').getHz();
+    this.metrics.setPropsTime = this.stats.get('setProps Time').time;
+    this.metrics.updateAttributesTime = this.stats.get('Update Attributes').time;
+    this.metrics.framesRedrawn = this.stats.get('Redraw Count').count;
+    this.metrics.pickTime =
+      this.stats.get('pickObject Time').time +
```

Comment:
Let's just merge these three stats. I don't think anyone is using one in particular.